dist/api_data.py

```diff
@@ -1776,18 +1775,20 @@ def __ge__(self, other):
 
 'WT_SESSION.commit_transaction' : Method([
     Config('commit_timestamp', '', r'''
-        set the commit timestamp for the current transaction. The supplied value
-        must not be older than the first commit timestamp already set for the
-        current transaction, if any. The value must also not be older than the
-        current oldest and stable timestamps. For prepared transactions, exactly one
-        commit timestamp is required; it must not be older than the prepare timestamp.
-        See @ref timestamp_txn_api'''),
+        set the commit timestamp for the current transaction. The
+        value must not be older than the first commit timestamp already set
+        for the current transaction, if any. The value must not be older than
+        the current oldest timestamp and must be after the current stable
+        timestamp. For prepared transactions, a commit timestamp is required,
+        must not be older than the prepare timestamp, can be set only once,
+        and must not be set until after the transaction has successfully
+        prepared. See @ref timestamp_txn_api and @ref timestamp_prepare'''),
     Config('durable_timestamp', '', r'''
-        set the durable timestamp for the current transaction. Required for the
-        commit of a prepared transaction, and otherwise not permitted. The
-        supplied value must not be older than the commit timestamp set for the
-        current transaction. The value must also be newer than the current
-        oldest and stable timestamps. See @ref timestamp_prepare'''),
+        set the durable timestamp for the current transaction. Required for
+        the commit of a prepared transaction, and otherwise not permitted. Can
+        only be set after the transaction has been prepared and a commit
+        timestamp has been set. The value must not be older than
+        the commit timestamp. See @ref timestamp_prepare'''),
     Config('operation_timeout_ms', '0', r'''
```

Comment:
This should still say that it must be newer than the current stable timestamp. (And oldest, but that's implied so doesn't matter one way or the other whether it's explicitly stated.)

Reply:
Fixed in 493d1dd.